schemas/context/profile-privacy.schema.json

```diff
@@ -0,0 +1,80 @@
+{
+  "meta:license": [
+		"Copyright 2017 Adobe Systems Incorporated. All rights reserved.",
+		"This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+		"you may not use this file except in compliance with the License. You may obtain a copy",
+		"of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/profile-privacy",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Profile Privacy",
+  "description": "Accepts Privacy Opt-Out signals for Selling/Sharing of data or general Opt-Outs by Users",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "definitions": {
+	"profile-privacy": {
+		"xdm:optOutConsentLevel": {
+			"type": "object",
+			"title": "Opt-Out consent level",
+			"description": "Opt-Out consent level",
```

Comment:
This description should describe what is to be captured in this field and should end with a period.

Reply:
This additional level is added as it was requested by profile team. there is no known property yet at this level. They need this to add properties at a higher level in future.